src/styles/mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin no-active-stars {
+  background: $no-active $background;
+}
+
+@mixin active-stars {
+  background: $active $background;
```

Comment:
Consider using styles from the previous task. The number of active stars should depend on the class with the modifier

Reply:
Thanks, done!